MapboxMobileEvents.podspec

```diff
@@ -0,0 +1,42 @@
+Pod::Spec.new do |s|
+
+  # ―――  Spec Metadata  ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+
+  s.name = 'MapboxMobileEvents'
+  s.version = '0.2.3'
+  s.summary = 'Mapbox Mobile Events'
+
+  s.description  = 'Mapbox Mobile Events'
+
+  s.homepage = 'https://www.mapbox.com/mobile/'
+
+  # ―――  Spec License  ――――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+
+  s.license = { :type => 'ISC', :file => 'LICENSE.md' }
+
+  # ――― Author Metadata  ――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+
+  s.author = { 'Mapbox' => 'mobile@mapbox.com' }
+  s.social_media_url = 'https://twitter.com/mapbox'
+
+  # ――― Platform Specifics ――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+
+  s.ios.deployment_target = '8.0'
+
+
+  # ――― Source Location ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+
+  s.source = { :git => 'https://github.com/mapbox/mapbox-telemetry-ios.git', :tag => 'v#{s.version.to_s}' }
+
+  # ――― Source Code ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+
+  s.source_files = ['MapboxMobileMetrics/MapboxMobileEvents/*', 'MapboxMobileMetrics/MapboxMobileEvents/Reachability/*']
```

Comment:
Use the globstar syntax (`**`) so you don’t have to remember to add additional paths in the future.